appcues/src/main/java/com/appcues/data/mapper/experience/ExperienceMapper.kt

```diff
@@ -51,23 +51,23 @@ internal class ExperienceMapper(
     }
 
     private fun List<StepContainerResponse>.mapToStepContainer(
-        superTraits: List<TraitResponse>,
+        superTraits: List<ExperienceTrait>,
         superActions: Map<UUID, List<ActionResponse>>?
     ) = map { stepContainerResponse ->
-        val mergedTraits = stepContainerResponse.traits.mergeTraits(superTraits)
-        val mappedTraits = traitsMapper.map(mergedTraits)
+        val mappedTraits = traitsMapper.map(stepContainerResponse.traits)
+        val mergedTraits = mappedTraits.mergeTraits(superTraits, EXPERIENCE)
```

Comment:
goes with my previous question i guess, but trying to understand why `EXPERIENCE` is used here when the traits are at the `GROUP` level for the container here.